### PR TITLE
Feature/api documentation

### PR DESCRIPTION
**Pull request by royanik17**
API DOCUMENTATION

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+- Member Admin Controller
+    - CREATE MEMBER
+  
+    - UPDATE MEMBER BY ID
+    
+    - DELETE MEMBER BY ID
+    
+    - INACTIVE MEMBER BY ID
+
+- CommunityAdminControllerV2
```
Comment (ayanit84) [inline README.md:24]: remove community

File: README.md [hunk 2]
```diff
@@ -0,0 +1,334 @@
+# membership-service
+Microservice based APIs to leverage different services towards community
+# API Documentation
+This README provides an overview and documentation for the Membership Service API. It describes the endpoints available, their functionalities, and how to use them.
+
+## Table of Contents
+    Endpoints
+
+ -  Member Search Controller
```
Comment (ayanit84) [inline README.md:9]: do not use "controller" word

File: README.md [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+
+# Endpoints 
+  ## Member Search Controller
+  
+### Get All Members
+  Retrieves a list of all members.
+
+  - URL:`/v2/members`
+  - Method: GET
+  - Response: `List<MemberResponseV2>`  - List of member details
```
Comment (ayanit84) [inline README.md:42]: List<Some Object> - no need to mention , just show the JSON structure

**Review by ayanit84 — CHANGES_REQUESTED** [review]
please check

File: README.md (hunk 1 above)
Comment (royanik17) [reply README.md]: Replace By API

File: README.md (hunk 2 above)
Comment (royanik17) [reply README.md]: API

File: README.md (hunk 3 above)
Comment (royanik17) [reply README.md]: Fixed by using JSON Structure

**Comment by royanik17** [comment]
@ayanit84  Updated you can review it now

**Review by ayanit84 — APPROVED** [review]
looks good.

**Pull request merged**